### PR TITLE
clean up subbed vars; add wing testing method

### PR DESCRIPTION
**Pull request by mjburton11**

### CODE REVIEW

**Comment by mjburton11** [comment]
test models please

**Comment by mjburton11** [comment]
@bqpd I added a testing method for the wing in `wing_test.py` how do we integrate testing with Jenkins for internal gpkitmodels testing?

**Comment by mjburton11** [comment]
test models please

File: gpkitmodels/GP/aircraft/wing/wing_test.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    W.substitutions[W.topvar("W")] = 50
+    fs = FlightState()
+    perf = W.flight_model(W, fs)
+    loading = W.loading(W, Wcent, W.topvar("W"), fs["V"], perf["C_L"])
+
+    m = Model(perf["C_d"], [W, fs, perf, loading])
+    m.solve("mosek")
+
+if __name__ == "__main__":
+    wing_test()
```
Comment (bqpd) [inline gpkitmodels/GP/aircraft/wing/wing_test.py]: Hey, our first gplibrary test. I should add this to jenkins.

Comment (mjburton11) [reply gpkitmodels/GP/aircraft/wing/wing_test.py]: OK perfect!

**Comment by mjburton11** [comment]
@galbramc I just pushed this branch and jenkins has put this build in the queue and it seems to be stuck there and I don't know how to get it out.

**Comment by galbramc** [comment]
Yeah I saw it had been stuck for 20 hours. No idea what happened. I just killed it and triggered a new build. Looks like it's running now.

**Comment by mjburton11** [comment]
odd, I tried to trigger a build about 20 hours ago, but it must have just got stuck in the queue.  And then when I pushed again this morning it must have thought it didn't need to start another build.  Is there any issue with my permissions maybe?

**Comment by mjburton11** [comment]
On my end, it doesn't look like anything is running...

**Comment by mjburton11** [comment]
just tried starting another build and it stuck me in the queue again.

**Comment by mjburton11** [comment]
@bqpd this also should have never passed the tests.  Because I deleted the input to `lam` in the wing model should it should have caused `jho`, `gas_solar_trade`, and `solar` to all fail...

**Comment by mjburton11** [comment]
See #128 for fix

**Comment by mjburton11** [comment]
@bqpd I think it's still testing all the repos on whatever branch is specified in `TESTCONFIG`, which is `master` for most of them.

**Pull request merged**